### DISCUSSION TITLE
sui-stack-overflow 基于walrus和sui构建的技术分享平台

### DISCUSSION THREAD

**Discussion by fantasy-move** [creatorsand/.github#139] [Ideas]
### 一句话 IDEA：
基于sui链和walrus搭建一个技术分享平台网站，用户在学习sui技术的同时，通过在网站上进行发帖，打赏等交互行为，能对sui链的基本数据流转有更好的了解

### 我的角色：
项目研发和产品

### 社群昵称：
桂振宇

### 个人简介：
 Java开发   任职于国内一家AI公司  
 技术栈:
       主要是Java开发
       python数据挖掘
       Next.js + Tailwind    

### 产品是给谁用的？

给想要了解和学习sui的用户

### 其他产品有什么痛点，用户为什么要选择你的产品？

无

**Comment by wufen771** [comment]
根据模版完善idea页面哦

**Comment by v1xingyue** [comment]
ai打分

[a comment by an automated account is not shown]
